py_mmd_tools/nc_to_mmd.py

```diff
@@ -988,18 +988,31 @@ def to_mmd(self, mmd_yaml=None, *args, **kwargs):
             self.metadata[key] = self.get_acdd_metadata(mmd_yaml[key], ncin, key)
 
         # Set storage_information
-        md5hasher = FileHash('md5', chunk_size=1048576)
-        fchecksum = md5hasher.hash_file(file_for_checksum_calculation)
+
         file_location = self.netcdf_product
-        self.metadata['storage_information'] = {
-            'file_name': os.path.basename(self.netcdf_product),
-            'file_location': file_location,
-            'file_format': 'NetCDF-CF',
-            'file_size': '%.2f'%file_size,
-            'file_size_unit': 'MB',
-            'checksum': fchecksum,
-            'checksum_type': '%ssum'%md5hasher.hash_algorithm,
-        }
+
+        if checksum_calculation:
+            md5hasher = FileHash('md5', chunk_size=1048576)
+            fchecksum = md5hasher.hash_file(file_for_checksum_calculation)
+
+            self.metadata['storage_information'] = {
+                'file_name': os.path.basename(self.netcdf_product),
+                'file_location': file_location,
+                'file_format': 'NetCDF-CF',
+                'file_size': '%.2f'%file_size,
+                'file_size_unit': 'MB',
+                'checksum': fchecksum,
+                'checksum_type': '%ssum'%md5hasher.hash_algorithm,
+            }
+
+        else:
+            self.metadata['storage_information'] = {
+                'file_name': os.path.basename(self.netcdf_product),
+                'file_location': file_location,
+                'file_format': 'NetCDF-CF',
+                'file_size': '%.2f'%file_size,
+                'file_size_unit': 'MB',
+            }
```

Comment:
```suggestion
        self.metadata['storage_information'] = {
                'file_name': os.path.basename(self.netcdf_product),
                'file_location': file_location,
                'file_format': 'NetCDF-CF',
                'file_size': '%.2f'%file_size,
                'file_size_unit': 'MB',
            }
        if checksum_calculation:
            md5hasher = FileHash('md5', chunk_size=1048576)
            fchecksum = md5hasher.hash_file(file_for_checksum_calculation)

            self.metadata['storage_information']['checksum'] = fchecksum
            self.metadata['storage_information']['checksum_type'] = '%ssum' % md5hasher.hash_algorithm
```

Reply:
it's also a bit strange with `%ssum` in `self.metadata['storage_information']['checksum_type'] = '%ssum' % md5hasher.hash_algorithm` - should the `sum` part be removed?